### DISCUSSION TITLE
Gurobipy support

### DISCUSSION THREAD

**Discussion by sharabhshukla** [ANL-CEEESA/MIPLearn#6] [Feature Requests]
Are there any planned support planned for gurobipy ?

**Comment by iSoron** [comment]
Hi @sharabhshukla, we plan to support `gurobipy` eventually, but we are focusing on Pyomo and JuMP right now. We actually already have a gurobipy implementation, but it is not documented at this time, and may change significantly.

**Comment by sharabhshukla** [comment]
Ok, thanks for the update. Keep up the good work!!
